src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="main">
+      <div class="main__card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="imac-image"
+          class="main__img"
+        >
+        <div class="main__card-content">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
```

Comment:
```suggestion
        <p class="main__card-content">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
```
Let's use more semantic tags instead of `div`